R-package/R/xgb.cv.R

```diff
@@ -196,7 +196,7 @@ xgb.cv <- function(params=list(), data, nrounds, nfold, label = NULL, missing =
         
         # early_Stopping
         if (!is.null(early.stop.round)){
-            score = strsplit(ret,'\\s+')[[1]][1+length(metrics)+2]
+            score = strsplit(ret,'\\s+')[[1]][1+length(metrics)+1]
```

Comment:
@tonglu thanks.  Yeah, I'm aware of this.  Haven't gotten around to fixing it yet.